### ISSUE TITLE
  Error: internal error, using unrecognized primitive #raw_function

### ISSUE THREAD

**Issue by a-c-sreedhar-reddy** [alexfedoseev/re-dnd#22]
I have installed `re-dnd` and added it to bsconfig.json. Then when I compile my project I get 

```sh
[141/255] Building src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.cmj
FAILED: src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.cmj src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.cmi /home/a-c-sreedhar-reddy/***/demo/node_modules/bs-webapi/src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.js 
/home/a-c-sreedhar-reddy/***/demo/node_modules/bs-platform/linux/bsc.exe -color always -bs-package-name bs-webapi  -bs-package-output es6:src/Webapi/Webapi__Dom -I src/Webapi/Webapi__ResizeObserver -I src/Webapi/Webapi__Dom -I src/Webapi/Webapi__Canvas -I src/Webapi -I src -w a -bs-no-version-header -bs-super-errors -o src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.cmj src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.reast

  We've found a bug for you!
  /home/a-c-sreedhar-reddy/***/demo/node_modules/bs-webapi/src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.re 11:5-7
  
   9 │ let set = (key, value, map) => set(map, key, value);
  10 │ let unsafeDeleteKey: (key, t) => unit =  
  11 │   [%raw (key, map) => "delete map[key];"];
  12 │   
  
  Error: internal error, using unrecognized primitive #raw_function
  
[146/255] Building src/Webapi/Webapi__Dom/Webapi__Dom__EventTarget.cmj
FAILED: subcommand failed.
Failure: /home/a-c-sreedhar-reddy/***/demo/node_modules/bs-platform/linux/ninja.exe 
 Location: /home/a-c-sreedhar-reddy/***/demo/node_modules/bs-webapi/lib/bs
```

**Comment by a-c-sreedhar-reddy** [comment]
This is fixed in this commit https://github.com/reasonml-community/bs-webapi-incubator/pull/184/files.

Can the bs-webapi be upgraded to the latest version?

**Comment by alex35mil** [comment]
I will do this week. Thanks for the report!

**Comment by alex35mil** [comment]
Fixed in #23
